lib/puppet/util/yaml.rb

```diff
@@ -2,13 +2,50 @@
 
 module Puppet::Util::Yaml
   if defined?(::Psych::SyntaxError)
```

Comment:
Good catch, those should definitely be insync. It looks like syck was completely removed in ruby 2, though can be [installed separately as a gem](https://github.com/ruby/ruby/blob/v2_3_0/lib/yaml.rb#L48-L49). That said syck doesn't implement `YAML.safe_load`, so we really do need/want Psych. Given https://github.com/ruby/ruby/blob/v2_3_0/lib/yaml.rb#L14 I think it's safe to assume `Psych` will always exist, and remove the `if defined?` check. Thoughts?

Reply:
Yeah, that seems reasonable, especially given the block just before the one you linked to: https://github.com/ruby/ruby/blob/v2_3_0/lib/yaml.rb#L5-L12 Ruby is going to complain _loudly_ if it can't load Psych.

Reply:
Fixed in commit ee85bdca84110bfac5000110083b69b2e4855bce